upgrade/upd_2.5.11-to-2.7.0/index.php

```
@@ -607,7 +613,164 @@ public function apply_deleteflashsanitizer(): bool
+        }
+
+        // Step 2: Recreate the unique index with an explicit prefix of 64.
+        if ($this->indexExists($table, 'field_name')) {
+            if (!$this->execOrFail("ALTER TABLE `{$table}` DROP INDEX `field_name`")) {
+                return false;
+            }
+        }
+        $addIndex = "ALTER TABLE `{$table}` ADD UNIQUE `field_name` (`field_name`(64)) USING BTREE";
+
+        return $this->execOrFail($addIndex);
+    }
```

PHPMD (2.15.0)

[error] The method apply_normalizeprofilefieldname is not named in camelCase. (undefined)

(CamelCaseMethodName)